### DISCUSSION TITLE
Complejidad IN-CIRCLE

### DISCUSSION THREAD

**Discussion by sofialarrain** [IIC2133-PUC/2024-1#97] [Tarea 1] [answered]
Hola! Tengo una pregunta sobre la función IN-CIRCLE de la parte 2. Si tengo un árbol que cumple con la complejidad de búsqueda promedio O(log n) y la búsqueda me construye un array con todas las cabezas que se encuentran dentro del círculo, pero estas no están ordenadas según x (y luego según y). Puedo utilizar un algoritmo de ordenamiento, como selection sort, para imprimir en orden la información de las cabezas?

**Comment by sebacarrascop** [comment, marked as answer]
holaa!, si puedes ya que tu complejidad aumentaría a lo más en klog(k), con k siendo despreciable en comparación a n, así que se seguiría manteniendo la complejidad de tu arbol.

**Reply by sofialarrain** [reply]
Perfecto! Gracias!
